docs/content/direct/examples.md

```diff
@@ -257,17 +255,17 @@ hosting cluster so that we can pass additional startup options.
 
 Label the `wds2` control plane as type `wds`:
 
-```shell 
+```shell
 kubectl label cp wds2 kflex.kubestellar.io/cptype=wds
 ```
 
 For this example, we use the `AppWrapper` custom resource defined in the
 [multi cluster app dispatcher](https://github.com/project-codeflare/multi-cluster-app-dispatcher)
 project.
 
-Install the AppWrapper CRD in the WDS and the WECs. Note that due to 
-[this issue](https://github.com/kubestellar/kubestellar/issues/1705) CRDs must be pre-installed 
-on the WDS and on the WECs when using API group filtering. For this release of KubeStellar, that pre-installation is required 
+Install the AppWrapper CRD in the WDS and the WECs. Note that due to
+[this issue](https://github.com/kubestellar/kubestellar/issues/1705) CRDs must be pre-installed
+on the WDS and on the WECs when using API group filtering. For this release of KubeStellar, that pre-installation is required
 when a WDS has a large number of API resources (such as would be found in a hosting cluster that is OpenShift).
```

Comment:
The whole "Note that... " sentence can be removed, that issue has been fixed.

Reply:
ok, removed